### ISSUE TITLE
Scss nth() calculation deprecation warnings.

### ISSUE THREAD

**Issue by RyannGalea** [webcat12345/ngx-ui-switch#439]
I'm using the latest 'ngx-ui-switch' in Angular 12.1, when compiling during development I get hit with about 100 of these per recompile.

```
[watch:angular] 58 │             padding-right: (nth($i, 2) / 3) * 1.5;
[watch:angular]    │                             ^^^^^^^^^^^^^^
[watch:angular]    ╵
[watch:angular]     node_modules\ngx-ui-switch\ui-switch.component.scss 58:29                   config-switch-sizes()
[watch:angular]     node_modules\ngx-ui-switch\ui-switch.component.scss 93:3                    @import
[watch:angular]     projects\outbuild\src\assets\styles\themes\default\_layout.module.scss 1:9  @import
[watch:angular]     projects\outbuild\src\assets\styles\themes\default\_modal.scss 1:9          @import
[watch:angular]     projects\outbuild\src\app\modals\auth\register\register.component.scss 1:9  root stylesheet
[watch:angular]
[watch:angular] DEPRECATION WARNING: Using / for division is deprecated and will be removed in Dart Sass 2.0.0.
[watch:angular]
[watch:angular] Recommendation: math.div(nth($i, 2), 3)
[watch:angular]
[watch:angular] More info and automated migrator: https://sass-lang.com/d/slash-div
```

Seems there is some sort of update for doing the calculations.

Ryann,

**Comment by cmckni3** [comment]
Yeah when using `sass` instead of `node-sass`, I saw this warning.

I couldn't get the library styles upgraded in a timely manner. The sass migrator should fix it automatically but I ran out of time.

PR welcome

**Comment by cmckni3** [comment]
Fixed in 13.0.2
